data/boosters/eoe-collector.yaml

```diff
@@ -0,0 +1,153 @@
+  stellar_sights: "e:EOS number:1-45"
+  poster_stellar_sights: "e:EOS number:46-90"
+  galaxy_stellar_sights: "e:EOS number:91-135"
+  galaxy_poster_stellar_sights: "e:EOS number:136-180"
+  celestial_basic: "e:{set} number:262-266"
+  galaxy_celestial_basic: "e:{set} number:367-371"
+  special_guest: "e:spg number:119-128"
+
+sheets:
+  foil_rare_mythic:
+    foil: true
+    any:
+    - use: rare
+      rate: 2
+    - use: mythic
+      rate: 1
+
+  celestial_basic_land:
+    foil: true
+    any:
+    - rawquery: "{celestial_basic}"
+      chance: 2
+      count: 5
+    - rawquery: "{galaxy_celestial_basic}"
+      chance: 1
+      count: 5
+
+  commander:
+    any:
+    - rawquery: "e:eoc r=m is:fullart number:1,2,3,4" # Full Art Commander Cards. For some reason the numbers need to be specified because it returns non fullart cards. If that gets fixed, "number:1,2,3,4" can be removed.
+      chance: 9
+      count: 4
+    - rawquery: "e:eoc r=r is:extendedart" # Extended Art Commmander Cards
+      chance: 91
+      count: 20
+
+  non_foil_boosterfun:
+    any:
+    - rawquery: "e:{set} r=r is:extendedart" # Extended Art Rare
+      chance: 47
+      count: 33
+    - rawquery: "e:{set} r=m is:extendedart" # Extended Art Mythic
+      chance: 5
+      count: 7
+    - rawquery: "{viewport} r=r" # Rare Viewport Land
+      chance: 7
+      count: 5
+    - rawquery: "{viewport} r=m" # Mythic Viewport Land
+      chance: 4
+    - rawquery: "{triumphant} r=r" # Rare Triumphant
+      chance: 17
+      count: 12
+    - rawquery: "{triumphant} r=m" # Mythic Trumphant
+      chance: 3
+      count: 4
+    - rawquery: "{surreal} r=r" # Rare Surreal
+      chance: 15
+      count: 11
+    - rawquery: "{surreal} r=m" # Mythic Surreal
+      chance: 2
+      count: 3
+
+  foil_boosterfun:
+    foil: true
+    any:
+    - rawquery: "e:{set} r=r is:extendedart" # Extended Art Rare
+      chance: 380
+    - rawquery: "e:{set} r=m is:extendedart" # Extended Art Mythic
+      chance: 40
+    - rawquery: "{viewport} r=r" # Rare Viewport Land
+      chance: 60
+    - rawquery: "{viewport} r=m" # Mythic Viewport Land
+      chance: 30
+    - rawquery: "{triumphant} r=r" # Rare Triumphant
+      chance: 130
+    - rawquery: "{triumphant} r=m" # Mythic Trumphant
+      chance: 20
+    - rawquery: "{surreal} r=r" # Rare Surreal
+      chance: 120
+    - rawquery: "{surreal} r=m" # Mythic Surreal
+      chance: 20
+    - rawquery: "{special_guest}" # Special Guest
+      chance: 60
+    - rawquery: "e:{set} number:357-366" # Japan Showcase
+      chance: 90
+    - rawquery: "e:{set} number: 383-392" # Fracture Foil Japan Showcase
+      chance: 10
+    - rawquery: "{galaxy_viewport} r=r" # Rare Galaxy Foil Viewport Land
+      chance: 20
+    - rawquery: "{galaxy_viewport} r=m" # Mythic Galaxy Foil Viewport Land
+      chance: 10
+    - rawquery: "e:{set} number=382" # Singularity Foil Sothera, The Supervoid
+      chance: 5 # Half as rare as mythic galaxy foil?
```

Comment:
I would expect the singularity foil sothera to be on the order of 1 of the mythic galaxy cards, not 2.5x as common.